FogCreek/Glitch-Community
# wind-blade: autofocus all inputs that are the first element inside of a popover

## Links
* https://wind-blade.glitch.me/

## Changes:
This autofocuses all text inputs that are the first element inside of a popover. Almost. The edit collection color text input is the second input, but I think we should focus the text input instead of the color input.

## How To Test:
* Open the "Create team" subpop from the user options pop
* Open the "Create collection" subpop from the "Add project to collection" pop on a project page
* Open the "Edit collection color" pop

^ The text input should be autofocused when you open each pop

## Feedback I'm looking for:
Does it work?
